Refit.Tests/InterfaceStubGenerator.cs

```diff
@@ -1,24 +1,50 @@
 ﻿using System;
 using System.Collections.Generic;
+using System.Collections.Immutable;
 using System.IO;
 using System.Linq;
 using System.Reflection;
+using System.Text;
 using System.Threading.Tasks;
 
 using Microsoft.CodeAnalysis;
 using Microsoft.CodeAnalysis.CSharp;
-using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Testing;
+using Microsoft.CodeAnalysis.Text;
 
-using Refit; // InterfaceStubGenerator looks for this
 using Refit.Generator;
+
 using Xunit;
 
 using Task = System.Threading.Tasks.Task;
+using VerifyCS = Refit.Tests.CSharpSourceGeneratorVerifier<Refit.Generator.InterfaceStubGenerator>;
 
 namespace Refit.Tests
 {
     public class InterfaceStubGeneratorTests
     {
+        static readonly MetadataReference RefitAssembly = MetadataReference.CreateFromFile(
+            typeof(GetAttribute).Assembly.Location,
+            documentation: XmlDocumentationProvider.CreateFromFile(Path.ChangeExtension(typeof(GetAttribute).Assembly.Location, ".xml")));
+
+        static readonly ReferenceAssemblies ReferenceAssemblies;
+
+        static InterfaceStubGeneratorTests()
+        {
+#if NET5_0_OR_GREATER
```

Comment:
🤦‍♂️ Looks like this was added for the 5.0.2xx SDK, but the CI is using 5.0.1xx.